src/controllers/handles.controller.ts

```diff
@@ -88,7 +92,7 @@ class HandlesController {
                 return;
             }
 
-            res.status(handleRepo.getIsCaughtUp() ? 200 : 202).json(handleData);
+            res.status(handleRepo.getIsCaughtUp() ? 200 : 202).json(new HandleViewModel(handleData));
```

Comment:
Should this also return a 404 if the utxo is missing?

Reply:
In the view model, if there is no UTxO it will throw an HttpError with a status of 404.